servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/NettyPipelinedConnectionTest.java

```diff
@@ -274,13 +274,13 @@ void readCancelErrorsPendingReadCancelsPendingWrite() {
         // should be terminated, cancelled, or not subscribed.
 
         assertThat(readSubscriber2.awaitOnError(), is(instanceOf(ClosedChannelException.class)));
-        writePublisher1Subscription.awaitCancelledUninterruptibly();
+        writePublisher1Subscription.awaitCancelled();
```

Comment:
I would also suggest to mark `awaitCancelledUninterruptibly()` as `@Deprecated`,  and reference that `awaitCancelled()` should be used instead. Since `awaitCancelledUninterruptibly()` is problematic for jUnit5, it will be good to remove that method in future releases.